oshi-core/src/main/java/oshi/driver/unix/openbsd/disk/Disklabel.java

```diff
@@ -0,0 +1,127 @@
+drivedata: 0
+
+*/
+    public static Pair<Map<String, String>, List<HWPartition>> getDiskParams(String diskName) {
+        List<HWPartition> partitions = new ArrayList<>();
+        HashMap<String, String> params = new HashMap<>();
+        Pair<Map<String, String>, List<HWPartition>> pair = new Pair<>(params, partitions);
+        // disklabel (requires root) supports 15 configurable partitions, `a' through `p', excluding `c'.
+        // The `c' partition describes the entire physical disk.
+        // By convention, the `a' partition of the boot disk is the root
+        // partition, and the `b' partition of the boot disk is the swap partition
+        // List<String> cmd = ExecutingCommand.runNative("doas disklabel -n " + diskName);
+        List<String> cmd = ExecutingCommand.runNative("disklabel -n " + diskName);
+        if (cmd.size() > 0) {
+            String[] split;
+            String line;
+            ListIterator<String> lines = cmd.listIterator();
+            while (lines.hasNext()) {
+                line = lines.next();
+                if (line.startsWith("16 partitions:")) {
+                    break;
+                }
+                split = line.split(":");
+                if (split.length == 2) {
+                    params.put(split[0], split[1].trim());
+                }
+            }
+            // should be just the partition table left
+            HWPartition p;
+            String name;
+            while (lines.hasNext()) {
+                line = lines.next();
+                //16 partitions:
+                //#                size           offset  fstype [fsize bsize   cpg]
+                //  a:          2097152             1024  4.2BSD   2048 16384 12958 # /
+                //  b:         17023368          2098176    swap                    # none
+                //  c:        500118192                0  unused
+                //  d:          8388576         19121568  4.2BSD   2048 16384 12958 # /tmp
+                //  e:         41386752         27510144  4.2BSD   2048 16384 12958 # /var
+                //  f:          4194304         68896896  4.2BSD   2048 16384 12958 # /usr
+                //  g:          2097152         73091200  4.2BSD   2048 16384 12958 # /usr/X11R6
+                //  h:         20971520         75188352  4.2BSD   2048 16384 12958 # /usr/local
+                //  i:              960               64   MSDOS
+                //  j:          4194304         96159872  4.2BSD   2048 16384 12958 # /usr/src
+                //  k:         12582912        100354176  4.2BSD   2048 16384 12958 # /usr/obj
+                //  l:        387166336        112937088  4.2BSD   4096 32768 26062 # /home
+                // Note size is in sectors
+                if (line.startsWith("#") || line.isEmpty()) {
+                    continue;
+                }
+                split = ParseUtil.whitespaces.split(line.trim(), 9);
+                name = split[0].replaceAll(":", "");
```

Comment:
Style/performance preference: Both splitting and `replaceAll` use regex.  See above on what I think is a better option using `substring()`.